src/Certificate_Item.php

```diff
@@ -200,143 +202,121 @@ public function deleteItemByCertificatesAndItem($certificates_id, $items_id, $it
+                'certificates_id' => $instID,
+                'dropdown_params' => [
+                    'items_id_name'   => 'items_id',
                     'itemtypes'       => Certificate::getTypes(true),
                     'entity_restrict' => ($certificate->fields['is_recursive']
-                                      ? getSonsOf(
-                                          'glpi_entities',
-                                          $certificate->fields['entities_id']
-                                      )
-                                       : $certificate->fields['entities_id']),
+                        ? getSonsOf(
+                            'glpi_entities',
+                            $certificate->fields['entities_id']
+                        )
+                        : $certificate->fields['entities_id']),
                     'checkright'      => true,
                 ]
-            );
-            echo "</td><td colspan='2' class='center' class='tab_bg_1'>";
-            echo Html::hidden('certificates_id', ['value' => $instID]);
-            echo Html::submit(_x('button', 'Add'), ['name' => 'add']);
-            echo "</td></tr>";
-            echo "</table>";
-            Html::closeForm();
-            echo "</div>";
-        }
-
-        echo "<div class='spaced'>";
-        if ($canedit && $number) {
-            Html::openMassiveActionsForm('mass' . __CLASS__ . $rand);
-            $massiveactionparams = [];
-            Html::showMassiveActions($massiveactionparams);
-        }
-        echo "<table class='tab_cadre_fixe'>";
-        echo "<tr>";
-
-        if ($canedit && $number) {
-            echo "<th width='10'>" .
-            Html::getCheckAllAsCheckbox('mass' . __CLASS__ . $rand) . "</th>";
-        }
-
-        echo "<th>" . _sn('Type', 'Types', 1) . "</th>";
-        echo "<th>" . __s('Name') . "</th>";
-        if (Session::isMultiEntitiesMode()) {
-            echo "<th>" . htmlescape(Entity::getTypeName(1)) . "</th>";
+            ];
+            // language=Twig
+            echo TemplateRenderer::getInstance()->renderFromStringTemplate(<<<TWIG
+                {% import 'components/form/fields_macros.html.twig' as fields %}
+                <div class="mb-3">
+                    <form method="post" name="certificates_form" action="{{ 'Certificate_Item'|itemtype_form_path }}">
+                        <input type="hidden" name="certificates_id" value="{{ certificates_id }}">
+                        <input type="hidden" name="_glpi_csrf_token" value="{{ csrf_token() }}">
+                        <div class="d-flex">
+                            {{ fields.dropdownItemsFromItemtypes('items_id', '', dropdown_params|merge({
+                                no_label: true,
+                                inline_add_field_html: true,
+                                add_field_html: '<button type="submit" name="add" class="btn btn-primary ms-3">' ~ btn_label ~ '</button>'
```

Comment:
Maybe it is possible to use a `field_class: 'd-inline'` option and move the `<button>` related code outside the field itself.

Having to add the button in the `add_field_html` option in order to get the expected rendering seems a bit complicated to me. We have to find a simple solution for this.